third_party/nvfuser/csrc/fusion.h

```diff
@@ -238,11 +239,24 @@ class TORCH_CUDA_CU_API Fusion : public IrContainer {
     return io_alias_;
   }
 
+  auto activeMultiGroupFusionBuilder() const {
```

Comment:
These three methods could be removed after changing what I suggest in ir_builder.cpp

Reply:
same for `active_multi_group_builder_ ` below

Reply:
removed